scripts/plaid.conf.js

```diff
@@ -36,9 +36,6 @@ exports.pages = {
     entry: './src/popup',
     html: htmlFactory(),
   },
-  injected: {
-    entry: './src/injected',
-  },
 };
 
 const splitVendor = name => ({
```

Comment:
`test` in `splitVendor` should be stricter, by adding trailing `/`.

Reply:
Why? I intentionally omit it to include all related repos e.g. lots of vue*.

Reply:
OK, then I prefer `prefix` rather than `name` as the parameter. I noticed this because a small CSS file `public/lib/vue.css` is generated and I thought it's unnecessary.

Reply:
Yep, renamed and added restriction on `.js`.